Exiled.API/Enums/ConfigType.cs

```diff
@@ -7,19 +7,27 @@
 
 namespace Exiled.API.Enums
 {
+    using System;
+
     /// <summary>
     /// The different types of configuration files distribution.
     /// </summary>
     public enum ConfigType
     {
         /// <summary>
-        /// Default distribution, every plugin will share the same config file.
+        /// Default. Removed in the future.
         /// </summary>
-        Default,
+        [Obsolete("Separated & Merged exist now.")]
```

Comment:
Why marking it as obsolete when we can just remove it?

Reply:
Possibly to not breaking EVERY SCP:SL config server ?

Reply:
Then just keep it as is without marking it obsolete.
Since we're going to break all plugins anyway, and people have time to migrate from v8 to v9, it looks fine removing it without using `Obsolete`.

Reply:
> Since we're going to break all plugins anyway, and people have time to migrate from v8 to v9, it looks fine removing it without using `Obsolete`.

why would we keep it as not Obsolete if we deleted it after ?